netdata/netdata
# bump go.d.plugin to v0.37.1

##### Summary

Fixes: #13549
Closes: #13550

bump go.d.plugin version to [v0.37.1](https://github.com/netdata/go.d.plugin/releases/tag/v0.37.1)

v0.37.1 contains a patch that changes the following charts:
- [galera_cluster_status](https://github.com/netdata/go.d.plugin/blob/faa01a27a40818eeef5b849ae599353ab000dc48/modules/mysql/charts.go#L751-L759) => [new version](https://github.com/netdata/go.d.plugin/blob/84fb4f433b1d9b2556361e2bdab2efac4367b424/modules/mysql/charts.go#L751-L762).
- [galera_cluster_state](https://github.com/netdata/go.d.plugin/blob/faa01a27a40818eeef5b849ae599353ab000dc48/modules/mysql/charts.go#L762-L771) => [new version](https://github.com/netdata/go.d.plugin/blob/84fb4f433b1d9b2556361e2bdab2efac4367b424/modules/mysql/charts.go#L764-L778).

TL;DR The change is switching from encoding the state/status with an integer value to a dimension per state/status.

In addition, this PR adjusts corresponding MySQL alarms and dashboard info.

##### Component Name

`packaging/`

##### Test Plan

- install this branch, ensure there are no errors, and go.d.plugin version is 0.37.1.
- check Galera charts and alarms.

##### Additional Information
